R/zchunk_L242.Bld_Inc_Elas_scenarios.R

```diff
@@ -1,88 +1,181 @@
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("common/GCAM_region_names", "socioeconomics/A42.inc_elas",
+                     "L101.Pop_thous_GCAM3_R_Y","temp-data-inject/L102.gdp_mil90usd_GCAM3_R_Y") %>%
+      add_flags() ->
       L242.IncomeElasticity_bld_GCAM3
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
-      add_legacy_name("object") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
-      object
-
-    return_data(L242.IncomeElasticity_bld_GCAM3, object)
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["gSSP1"]] %>%
+      add_title("Building Income Elasticity: gSSP1") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_gSSP1") ->
+      L242.IncomeElasticity_bld_gSSP1
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["gSSP2"]] %>%
+      add_title("Building Income Elasticity: gSSP2") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_gSSP2") ->
+      L242.IncomeElasticity_bld_gSSP2
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["gSSP3"]] %>%
+      add_title("Building Income Elasticity: gSSP3") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_gSSP3") ->
+      L242.IncomeElasticity_bld_gSSP3
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["gSSP4"]] %>%
+      add_title("Building Income Elasticity: gSSP4") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_gSSP4") ->
+      L242.IncomeElasticity_bld_gSSP4
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["gSSP5"]] %>%
+      add_title("Building Income Elasticity: gSSP5") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_gSSP5") ->
+      L242.IncomeElasticity_bld_gSSP5
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["SSP1"]] %>%
+      add_title("Building Income Elasticity: SSP1") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_SSP1") ->
+      L242.IncomeElasticity_bld_SSP1
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["SSP2"]] %>%
+      add_title("Building Income Elasticity: SSP2") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_SSP2") ->
+      L242.IncomeElasticity_bld_SSP2
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["SSP3"]] %>%
+      add_title("Building Income Elasticity: SSP3") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_SSP3") ->
+      L242.IncomeElasticity_bld_SSP3
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["SSP4"]] %>%
+      add_title("Building Income Elasticity: SSP4") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_SSP4") ->
+      L242.IncomeElasticity_bld_SSP4
+
+    L242.pcgdp_thous90USD_Scen_R_Y[["SSP5"]] %>%
+      add_title("Building Income Elasticity: SSP5") %>%
+      add_legacy_name("L242.IncomeElasticity_bld_SSP5") ->
+      L242.IncomeElasticity_bld_SSP5
+
+    return_data(L242.IncomeElasticity_bld_GCAM3,
```

Comment:
As Haewon pointed out, we are no longer building the GCAM3 scenarios, so this output should be removed, along with all of the code to build it.